Sources/HiPMobile/ServiceAccessLayer/AuthenticationApiAccess/AuthApiAccess.cs

```diff
@@ -40,7 +44,8 @@ public async Task<Token> Login(string username, string password)
 
             if (result.StatusCode == HttpStatusCode.BadRequest || result.StatusCode == HttpStatusCode.NotFound || result.StatusCode == HttpStatusCode.Forbidden)
             {
-                throw new InvalidUserNamePassword();
+                //throw new InvalidUserNamePassword();
```

Comment:
Why keep the comments? Same in line 33.

Reply:
removing the comments... I had commented the lines just to remember what I had changed when I was implementing it....